internal/common/grpc/grpcpool/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I really don't know what to do about this.  

I believe Armada-Operator is Licensed as Apache.  How does mixing different Software licenses work?

Reply:
https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project#:~:text=Both%20the%20Apache%20License%20and,as%20required%20by%20the%20license.

Reply:
well TIL.  

I guess its fine then?

Reply:
Ship it.